bigtop-packages/src/common/flink/flink-master.svc

```diff
@@ -0,0 +1,88 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TYPE="master"
+DAEMON="flink-${TYPE}"
+DESC="Flink ${TYPE}"
+EXEC_PATH="/usr/lib/flink/bin/flink-daemon.sh"
+WORKING_DIR="/var/lib/flink"
+DAEMON_FLAGS=jobmanager
+CONF_DIR="/etc/flink/conf"
+SVC_USER="root"
+PIDFILE="/var/run/flink/${DAEMON}.pid"
+EXEC_MODE="cluster"
+
+generate_start() {
+ 
+cat <<'__EOT__'
+start() {
+    [ -x $EXE_FILE ] || exit $ERROR_PROGRAM_NOT_INSTALLED
+    log_success_msg "Starting $DESC (${DAEMON}): "
+
+    checkstatusofproc
+    status=$?
+    if [ "$status" -eq "$STATUS_RUNNING" ]; then
+        log_success_msg "${DESC} is running"
+        exit 0
+    fi
+
+    LOG_FILE=/var/log/flink/${DAEMON}.out
+
+    if [ -f $CONF_DIR/flink-env.sh ]; then
+        . $CONF_DIR/flink-env.sh
+    fi
+
+    if [ "$FLINK_MASTER_IP" = "" ]; then
+        FLINK_MASTER_IP=`hostname`
```

Comment:
I think this can be removed safely. This was set for testing purpose, to provide the workers with the master-node name. I don't think we need it now.